multidict/_multidict.c

```diff
@@ -589,7 +589,7 @@ multidict_tp_dealloc(MultiDictObject *self)
         PyObject_ClearWeakRefs((PyObject *)self);
     };
     pair_list_dealloc(&self->pairs);
-    PyObject_GC_Del(self);
+    Py_TYPE(self)->tp_free((PyObject *)self);
```

Comment:
What about adding macro `#define MultiDictTpFree(ob) (Py_TYPE(ob)->tp_free((PyObject *)ob))` ?

Reply:
This macro will be used at most twice: once for MultiDict/CIMultiDict and once for their proxies.
Do we really need the macro for such a case?
The expanded version is better readable from my perspective.

Reply:
Ok, you right.